pkg/chartutil/errors.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package chartutil
+
+import (
+	"fmt"
+)
+
+var (
+	// ErrChartMetadataMissing is raised when a Chart.yaml is missing from a chart.
```

Comment:
Isn't that's what is happening here? e.g. in load_test.go it is doing type assertion that the error is of type ErrChartMetadataMissing down below. :) it's just a matter of downstream packages of pkg/chartutil to use these error types moving forward.